### PR TITLE
Implemented CustomBlock.onInteract() and CustomBlock.onClick()

### PR DESCRIPTION
**Pull request by Pamelloes**
These methods used to work but have stopped working after I updated a few days ago. If the methods are called elsewhere, please tell me so that I can fix the methods' implementations there.

### CODE REVIEW

File: src/main/java/org/getspout/spout/SpoutPlayerListener.java [hunk 1]
```diff
@@ -121,8 +121,14 @@ public void onPlayerInteract(PlayerInteractEvent event) {
 			return;
 		}
 		if (event.getAction() != Action.RIGHT_CLICK_BLOCK) {
+			if(event.getAction() == Action.LEFT_CLICK_BLOCK) {
+				SpoutBlock block = (SpoutBlock)event.getClickedBlock();
+				if(block.isCustomBlock()) block.getCustomBlock().onBlockClicked(block.getWorld(), block.getX(), block.getY(), block.getZ(), (SpoutPlayer) event.getPlayer());
+			}
 			return;
 		}
+		if (!action) {
```
Comment (Top-Cat) [inline src/main/java/org/getspout/spout/SpoutPlayerListener.java]: erm... what?

Comment (Pamelloes) [inline src/main/java/org/getspout/spout/SpoutPlayerListener.java]: erm...... no idea how that got there :D
I'll fix it in a few secs

Comment (Pamelloes) [inline src/main/java/org/getspout/spout/SpoutPlayerListener.java]: Fixed.

**Comment by Pamelloes** [comment]
Actually, don't accept this PR yet - I need to do a little more testing.

**Pull request closed without merging**